src/components/AvatarWithDisplayName.js

```diff
@@ -41,16 +44,32 @@ const defaultProps = {
     personalDetails: {},
     policies: {},
     report: null,
+    isAnonymous: false,
     size: CONST.AVATAR_SIZE.DEFAULT,
 };
 
 const AvatarWithDisplayName = (props) => {
-    const title = ReportUtils.getDisplayNameForParticipant(props.report.ownerEmail, true);
+    const title = props.isAnonymous ? props.report.displayName : ReportUtils.getDisplayNameForParticipant(props.report.ownerEmail, true);
```

Comment:
Prop is defaulting to `null` so this might throw an error because we're trying to access `displayName` on a null reference

Reply:
Ah that's true, updating the report prop